eox_core/api/v1/views.py

```diff
@@ -581,6 +582,11 @@ def post(self, request, *args, **kwargs):
         - `is_active` (boolean, _body_):
             Flag indicating whether the enrollment is active.
 
+        - `force` (boolean, _body_):
+            Flag indicating whether the platform business rules for enrollment must be skipped. When it is true, the enrollment
+            is created without looking at the enrollment dates, if the course is full, if the enrollment mode is part of the modes
```

Comment:
are you sure the course mode is not checked? https://github.com/eduNEXT/eox-core/blob/a6d976a6cec34a64cbc028912030f8c5fc34aa03/eox_core/edxapp_wrapper/backends/enrollment_h_v1.py#L207

Reply:
yes, this function checks that the mode belongs to the defined modes, but if it is forced it skips the check if a mode belongs to a course.
https://github.com/eduNEXT/eox-core/blob/a6d976a6cec34a64cbc028912030f8c5fc34aa03/eox_core/edxapp_wrapper/backends/enrollment_h_v1.py#L316-L318

Reply:
what about this line? https://github.com/eduNEXT/eox-core/blob/a6d976a6cec34a64cbc028912030f8c5fc34aa03/eox_core/edxapp_wrapper/backends/enrollment_h_v1.py#L311